index.js

```diff
@@ -162,35 +165,37 @@ exports.init = function (sbot, config) {
     Legacy(sbot, ebt)
   }
 
-  return {
-    replicate: function (opts) {
-      if (opts.version !== 2 && opts.version !== 3) {
-        throw new Error('expected ebt.replicate({version: 3 or 2})')
-      }
-      return toPull.duplex(ebt.createStream(this.id, opts.version, false))
-    },
-    // get replication status for feeds for this id.
-    peerStatus: function (id) {
-      id = id || sbot.id
-      const data = {
-        id: id,
-        seq: ebt.state.clock[id],
-        peers: {}
-      }
-      for (const k in ebt.state.peers) {
-        const peer = ebt.state.peers[k]
-        if (peer.clock[id] != null || peer.replicating[id] != null) {
-          const rep = peer.replicating && peer.replicating[id]
-          data.peers[k] = {
-            seq: peer.clock[id],
-            replicating: rep
-          }
+  function replicate (opts) {
+    if (opts.version !== 2 && opts.version !== 3) {
```

Comment:
this check can probably be changed to only version 3, because version 2 support was removed in EBT 8.

Reply:
Good to know. I'll do it in another PR since this one is just cosmetics